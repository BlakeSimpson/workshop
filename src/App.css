.App {
  text-align: center;
  background-image: url('https://upload.wikimedia.org/wikipedia/commons/7/71/Surfer_Portrait_(12645956723).jpg');
  background-size: cover;
  height: 100%;
  width: 100%;
  padding: 20px;
  color: white;
  display: flex;
  align-items: center;
  justify-content: center;
  position: relative;
}

.App__overlay {
  width: 100%;
  height: 100%;
  position: absolute;
  top: 0;
  left: 0;
  background-color: rgba(246, 59, 30, 0.5)
}

.App h1 {
  position: relative;
  text-shadow: 0px 1px #560e03;
}
