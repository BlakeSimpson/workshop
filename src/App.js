import React from 'react';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import TodoList from './TodoList';
import mainReducer from './reducer';

const store = createStore(mainReducer);

const App = () => {
	return (
    <Provider store={store}>
		  <TodoList />
    </Provider>
	)
}

export default App;
