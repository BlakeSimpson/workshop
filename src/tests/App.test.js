import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from '../App';

it('should render the react-redux Provider with a store and the TodoList', () => {
	const tree = toJson(shallow(<App />));
	expect(tree).toMatchSnapshot();
});
