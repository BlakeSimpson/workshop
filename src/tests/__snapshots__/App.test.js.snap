exports[`test should render the react-redux Provider with a store and the TodoList 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }>
  <Connect(TodoList) />
</Provider>
`;
